components/02-molecules/video/video.twig

```diff
@@ -0,0 +1,39 @@
+{#
+ # Available variables:
+ # - text_with_video__heading
+ # - text_with_video__text
+ #
+ # Available blocks:
+ # - text_with_video__video
+ #}
```

Comment:
We forgot to update the variable names in this file. They should all be something like `video__base_class` instead of `text_with_video__base_class`